spyder/config/manager.py

```diff
@@ -729,12 +755,18 @@ def reset_shortcuts(self):
             # TODO: check if the section exists?
             plugin_config.reset_to_defaults(section='shortcuts')
 
+        # This necessary to notify the observers of widget shortcuts
+        self.notify_section_all_observers(section="shortcuts")
+
 
 try:
     CONF = ConfigurationManager()
 except Exception:
     from qtpy.QtWidgets import QApplication, QMessageBox
 
+    # Print traceback to show error in the terminal in case it's needed
+    print(traceback.format_exc())  # spyder: test-skip
```

Comment:
Shouldn't we use the logging module for this?

Reply:
I don't think so because this will help third-party developers to see, even without logging enabled, that their plugin config is incorrect, like this:

![imagen](https://github.com/user-attachments/assets/a9785f34-17ab-40fa-8b67-67ab8e314d7f)